ui/views/tx/submit/horizon-error-handler.js

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
Let's move this logic to the separate package, e.g. `@stellar-expert/transaction-error-handler`